templates/details/configure.html

```diff
@@ -41,4 +42,33 @@
     {% endif %}
   </div>
 </div>
+
+<style>
+  .section-marker {
+    display: block;
+    margin-top: -64px;
+    padding-bottom: 64px;
+  }
+</style>
+
+<script>
+  const sideNavigationLinks = Array.prototype.slice.call(
```

Comment:
I'm pretty sure that every browser that supports arrow functions also supports `forEach` on a `NodeList`.